lib/task/taxonomy/taxonomyNormalizeTask.class.php

```diff
@@ -67,19 +67,21 @@ protected function execute($arguments = array(), $options = array())
   {
     parent::execute($arguments, $options);
 
-    $this->log("Normalizing for '". $options['culture'] ."' culture...");
-
     // Look up taxonomy ID using name
     $taxonomyId = $this->getTaxonomyIdByName($arguments['taxonomy-name'], $options['culture']);
     if (!$taxonomyId)
     {
       throw new sfException("A taxonomy named '". $arguments['taxonomy-name'] ."' not found for culture '". $options['culture'] ."'.");
     }
 
+    $this->log("Normalizing for '". $options['culture'] ."' culture...");
```

Comment:
On second thought, concatenation's used throughout the script and might be more time than it's worth to change. :/